Sources/StreamChat/AudioPlayer/AudioPlaybackContext.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// A struct that represents the current state of an audio player
+public struct AudioPlaybackContext: Equatable {
+    /// The duration of the audio track in seconds
+    public var duration: TimeInterval
```

Comment:
Why are these public vars?

Reply:
`let` properties on classes make sense to allow for instance immutability. On structs, where immutability can be controlled at the caller's site, IMHO `let` properties are just limiting for the API and that's why i'm always starting with `var` and only changing to `let` when it's necessary.
That being said, i'm not strongly opinionated and if you think there are cases where allowing this struct to be mutable that can hurt our flows i'm happy to change the properties to `let` in a subsequent PR.